.github/workflows/flake8.yml

```diff
@@ -0,0 +1,59 @@
+name: Python code quality check
+
+on:
+- push
+- pull_request
+
+jobs:
+  flake8-grass7-scripts:
+    runs-on: ubuntu-20.04
+
+    cd grass7/${{ matrix.directory }}
```

Comment:
I don't think this is supposed to work. I don't see the jobs running either.

Reply:
Well, the new attempt starts but fails miserably. No idea how it is written properly...

Reply:
Fixes added...